olm-post-script.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# This file is part of MinIO DirectPV
+# Copyright (c) 2023 MinIO, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+### General Objective:
+#
+# To add additional things once images has been pushed in release-operator.sh
+#
+### Reasoning:
+#
+# We can't add these additional things until images are pushed.
+# This is why we need a post script similar to what we are doing at
+# Operator repo: https://github.com/minio/operator/blob/master/olm-post-script.sh
+
+### relatedImages: Field needed by RedHat Certification.
+# kind: ClusterServiceVersion
+# spec:
+#   relatedImages:
+#     - image: gcr.io/kubebuilder/kube-rbac-proxy@sha256:<digest>
+#       name: kube-rbac-proxy
+#     - image: quay.io/minio/directpv-operator@sha256:<digest>
+#       name: manager
+
+PROXY_DIGEST=$(docker pull gcr.io/kubebuilder/kube-rbac-proxy:v0.13.1 | grep Digest | awk -F ' ' '{print $2}')
+OPERATOR_DIGEST=$(docker pull quay.io/minio/directpv-operator:4.0.7 | grep Digest | awk -F ' ' '{print $2}')
```

Comment:
Can we use `podman`  instead of docker? Also can this script be merged into `release-operator.sh`?

Reply:
Yes @balamurugana I will use `podman` and merge it to `release-operator.sh`. Thank you! 👍